### PR TITLE
LPS-175427 Adapt test in NotificationQueueEntryLocalServiceTest

### PR DESCRIPTION
**Pull request by devpedrotavares**
These commits were almost entirely approved in this [pull request](https://github.com/liferay-objects/liferay-portal/pull/1750).
The changes were from 59d8bf2bf0918541794eceff15c637439e94c26c to 723c46513e608cf627c91b998f433cdf1fdada16.
Obs.: the difference is that we are using a different name for the exception and that we're creating it through service builder.

### CODE REVIEW

File: modules/apps/notification/notification-service/src/main/java/com/liferay/notification/service/impl/NotificationQueueEntryLocalServiceImpl.java [hunk 1]
```diff
@@ -184,6 +185,16 @@ public NotificationQueueEntry resendNotificationQueueEntry(
 		NotificationQueueEntry notificationQueueEntry =
 			getNotificationQueueEntry(notificationQueueEntryId);
 
+		if (notificationQueueEntry.getStatus() ==
+				NotificationQueueEntryConstants.STATUS_SENT) {
+
+			throw new NotificationQueueEntryStatusException(
+				String.format(
+					"Must not resend notification with Id %d, because it has " +
```
Comment (marcelabc) [inline modules/apps/notification/notification-service/src/main/java/com/liferay/notification/service/impl/NotificationQueueEntryLocalServiceImpl.java:193]: Did you find examples of exception similar to this one? 
I found examples using the term ID. Could you change the word "Id" to "ID", please?

**Review by marcelabc — CHANGES_REQUESTED** [review]
Hi 
I finished my review. Let me know if you have questions :smile:

File: modules/apps/notification/notification-service/src/main/java/com/liferay/notification/service/impl/NotificationQueueEntryLocalServiceImpl.java (hunk 1 above)
Comment (devpedrotavares) [reply modules/apps/notification/notification-service/src/main/java/com/liferay/notification/service/impl/NotificationQueueEntryLocalServiceImpl.java]: You're right! Yes, I just did it

**Comment by devpedrotavares** [comment]
ci:test:object

**Review by marcelabc — APPROVED** [review]
LGTM 🎉

**Comment by marcelabc** [comment]
ci:test:sf

**Comment by devpedrotavares** [comment]
ci:test:sf

**Comment by devpedrotavares** [comment]
ci:test:relevant

**Comment by devpedrotavares** [comment]
ci:test:sf

**Comment by devpedrotavares** [comment]
ci:test:relevant

**Comment by devpedrotavares** [comment]
As discussed in the daily, I rebased it back on master. For now, we will wait for master to be synchronized with Brian's branch

**Comment by devpedrotavares** [comment]
ci:test:relevant

**Comment by devpedrotavares** [comment]
ci:test:sf

**Comment by devpedrotavares** [comment]
> As discussed in the daily, I rebased it back on master. For now, we will wait for master to be synchronized with Brian's branch. So that we can fix the conflict and send it with success on SF.

master was just synchronized with Brian, so I rebased and fixed the conflict (it was just a feature flag in `portal.properties`). Since sf passed, the object suite had already been approved by QA and the relevant failed don't seem related, I'm forwarding directly to Brian: https://github.com/brianchandotcom/liferay-portal/pull/131504

**Comment by devpedrotavares** [comment]
ci:close

**Pull request closed without merging**